client/landing/gutenboarding/onboarding-block/acquire-intent/style.scss

```diff
@@ -83,3 +91,21 @@ $acquire-intent-vertical-transition-algorithm: ease-in-out;
 		box-shadow: none;
 	}
 }
+
+.site-title {
+	transition: opacity $acquire-intent-transition-duration $acquire-intent-transition-algorithm;
+
+	&--without-value {
+		.madlib__input {
+			width: 200px;
+			display: inline-block;
+			vertical-align: middle;
+			border: none;
+		}
+	}
+
+	&--hidden {
+		visibility: hidden;
```

Comment:
The transition will work only in one direction this way, only when appearing. When disappearing, it will disappear abruptly, if that's the intention  👍 
```suggestion
		
```

Reply:
Yes, just fading in is the intention and also the `visibility: hidden` in this case is one of the solutions that prevents the user from accidentally selecting and entering some text in the Site Title field.